pkgs/tools/admin/fastlane/default.nix

```diff
@@ -0,0 +1,16 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  inherit ruby;
+  pname = "fastlane";
+  gemdir = ./.;
+
+  meta = with lib; {
+    description = "fastlane is a tool for iOS and Android developers to automate tedious tasks like generating screenshots, dealing with provisioning profiles, and releasing your application.";
```

Comment:
Description should not start with the package name and should probably be shorter. Maybe something like `"A tool to automate building and releasing iOS and Android apps"`? (You can set `longDescription` for longer descriptions if you'd like.)